_site/documentation/index.html

```diff
@@ -1 +1 @@
e collapsed" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1" aria-expanded="false"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar"></span> <span class="icon-bar"></span> <span class="icon-bar"></span> </button> <a class="navbar-brand" href="/emukit/">Emukit</a> </div> <!-- Collect the nav links, forms, and other content for toggling --> <div class="collapse navbar-collapse " id="bs-example-navbar-collapse-1"> <ul class="nav navbar-nav navbar-right"> <li><a href="/emukit/about/">About</a></li> <li><a href="/emukit/news/">News</a></li> <li class="dropdown"> <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Resources<span class="caret"></span></a> <ul class="dropdown-menu"> <li><a href="/emukit/installation/">Installation</a></li> <li><a href="/emukit/fiststeps/">First Steps</a></li> <li role="separator" class="divider"></li> <li><a href="/emukit/manual/">Reference Manual</a></li> <li><a href="/emukit/documentation/">Documentation</a></li> <li role="separator" class="divider"></li> <li><a href="/emukit/citation/">Citation</a></li> </ul> <li><a href="https://github.com/amzn/emukit">Github</a></li> </li> </ul> </div><!-- /.navbar-collapse --> </div><!-- /.container-fluid --> </nav> </header> <div class="container"> <div class="wrapper"> <div class="row"> <div class="col-md-8"> <article class="post"> <header class="post-header"> <h1 class="post-title">Documentation</h1> </header> <div class="post-content"> </div> </article> </div> <div class="col-md-4 mt20"> <div class="post-img"> <img width="600" src="/emukit/images/emulator.png" alt="webjeda"> </div> <div class="mt10 recent"> <h2>Also in Emukit</h2> <ul> </ul> </div> </div> </div> </div> </div> <footer> <div class="container"> <div class="row p20"> <div class="col-md-4 text-center mt25"> <a target="_blank" href="http://twitter.com/@emukitUQ"><li class="social twitter"><i class="fa fa-twitter-square"></i></li></a> <a target="_blank" href="https://github.com/amzn/emukit"><li class="social github"><i class="fa fa-github-square"></i></li></a> <a target="_blank" href="mailto:emukit-mailist@amazon.com"><li class="social email"><i class="fa fa-envelope"></i></li></a> </div> </div> </div> </footer> <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script> <script src="/emukit/js/bootstrap.min.js"></script> <script type="text/javascript" src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script> <!-- Google Analytics Tracking code --> <script> (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){ (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o), m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m) })(window,document,'script','//www.google-analytics.com/analytics.js','ga'); ga('create', 'UA-83979019-1', 'auto'); ga('send', 'pageview'); </script> </body> </html>
```

Comment:
Any idea why these files get update on each and every PR?

Reply:
No idea, I guess it has to do with the way the web is regenerated every time a change is introduced